lib/src/node/compile.dart

```diff
@@ -40,7 +46,8 @@ NodeCompileResult compile(String path, [CompileOptions? options]) {
         sourceMap: options?.sourceMap ?? false,
         logger: NodeToDartLogger(options?.logger, Logger.stderr(color: color),
             ascii: ascii),
-        importers: options?.importers?.map(_parseImporter));
+        importers: options?.importers?.map(_parseImporter),
+        functions: _parseFunctions(options?.functions).cast());
```

Comment:
Does this need to be `cast`ed to a specific type?

Reply:
No, it'll automatically infer the type of the parameter to which it's being passed.